forgerock-auth-ui/src/main/java/org/forgerock/android/auth/ui/callback/AdaptiveCallbackFragment.java

```diff
@@ -77,6 +79,20 @@ public View onCreateView(@NonNull LayoutInflater inflater, ViewGroup container,
         Button nextButton = view.findViewById(R.id.next);
         Button cancelButton = view.findViewById(R.id.cancel);
 
+        TextView header = view.findViewById(R.id.header);
+        if (isEmpty(current.getHeader())) {
+            header.setVisibility(GONE);
```

Comment:
What is the effect of `GONE` here?

Reply:
With GONE, the UI element will be removed from the layout, so it won't take any space. There are other options INVISIBLE, VISIBLE. To make it INVISIBLE, the UI element is invisible but still takes some space from the layout.